modules/server/src/main/java/org/jboss/wsf/stack/cxf/configuration/BusHolder.java

```diff
@@ -398,8 +398,7 @@ protected static void setCXFManagement(Bus bus, Map<String, String> props) {
             InstrumentationManagerExtImpl instrumentationManagerImpl = new InstrumentationManagerExtImpl();
             instrumentationManagerImpl.setBus(bus);
             instrumentationManagerImpl.setEnabled(true);
-            instrumentationManagerImpl.initMBeanServer();
-            instrumentationManagerImpl.register();
```

Comment:
Thanks for the patch , Viral .  It looks like we still need to add instrumentationManagerImpl.init() here to get  instrumentationManagerImpl registered in bus.  I'll add this and commit.